bbot/modules/postman_download.py

```diff
@@ -178,6 +182,47 @@ async def get_collection(self, collection_id):
         collection = json.get("collection", {})
         return collection
 
+    async def validate_workspace(self, workspace, environments, collections):
```

Comment:
This can be simplified to:

```python
    async def validate_workspace(self, workspace, environments, collections):
        name = workspace.get("name", "")
        for host in await self.scan.extract_in_scope_hostnames(str([workspace, environments, collections])):
            self.verbose(
                f'Found in-scope hostname: "{host}" for workspace {name}, it appears to be in-scope'
            )
            return True
        return False
```

Reply:
Added `full_wks = str([workspace, environments, collections])` to concatenate all the dictionaries in a workspace into a single string

Reply:
Ah good point about the only in-scope makes the `if self.scan.in_scope(host):` redundant in this instance. 👍